app/lib/LevelSetupManager.coffee

```diff
@@ -100,13 +100,12 @@ module.exports = class LevelSetupManager extends CocoClass
     else if allowedHeroOriginals = @level.get 'allowedHeroes'
       unless _.contains allowedHeroOriginals, me.get('heroConfig')?.thangType
         firstModal = @heroesModal
-    firstModal = @inventoryModal if me.isStudent()
+     
```

Comment:
Why did we remove firstModal from here? I think it should be controlled by the classroomItems flag

Reply:
If the classroomItems flag is off, the workflow does not even come here. If the flag is ON, we don't want to do this. So, I removed the line.